testing/execute_e2e_tests.sh

```diff
@@ -30,6 +30,7 @@ docker exec -i dala-e2e-test-backend-db-1 /bin/bash -c "PGPASSWORD=${BACKEND_DB_
 docker exec -i dala-e2e-test-api-key-manager-db-1 /bin/bash -c "PGPASSWORD=${API_KEY_MANAGER_DB_PASSWORD} pg_dump --username api_key_manager api_key_manager" > ./dbdumps/${CYPRESS_TEST_GROUP}/api-key-manager-db.sql || true
 docker exec -i dala-e2e-test-internal-storage-db-1 /bin/bash -c "PGPASSWORD=${INTERNAL_STORAGE_DB_PASSWORD} pg_dump --username internal_storage internal_storage" > ./dbdumps/${CYPRESS_TEST_GROUP}/internal-storage-db.sql || true
 docker exec -i dala-e2e-test-document-manager-db-1 /bin/bash -c "PGPASSWORD=${DOCUMENT_MANAGER_DB_PASSWORD} pg_dump --username document_manager document_manager" > ./dbdumps/${CYPRESS_TEST_GROUP}/document-manager-db.sql || true
+docker exec -i dala-e2e-test-qa-service-db-1 /bin/bash -c "PGPASSWORD=${QA_SERVICE_DB_PASSWORD} pg_dump --username qa_service qa_service" > ./dbdumps/${CYPRESS_TEST_GROUP}/qa-service-db.sql || true
```

Comment:
I don't know. Maybe ask the rest of the team.

Reply:
Ok I will ask in the group chat.

Reply:
I asked Siegfried first. This is the outcome:

- Yes, if the the dumping process is broken, the CI will not fail and you might not notice this, because `|| true` makes the script continue.
- It is possible that the dumping process is broken for months, and noone has noticed so far because noone has tried to look into the dumps for months.
- If smth like that happens, we can still fix the dumping at that time.
- The dump does not contain valuable data, since this is a CI run
- There are ways to make us notice if the dumping process fails, but at this moment we don't invest time into this, becaue the reward is not very valuable to us ("noone really cares about the dumps")